src/Util.cpp

```diff
@@ -37,6 +37,31 @@ std::string getImageId(json isd) {
   return id;
 }
 
+
+std::vector<double> getGeoTransform(json isd) {
+  std::vector<double> transform = {};
+  try {
+    transform = isd.at("geotransform").get<std::vector<double>>();
+  } catch (std::exception &e) {
+    std::string originalError = e.what();
+    std::string msg = "Could not parse the geo_transform. ERROR: " + originalError + isd.dump();
```

Comment:
Might want a new line character between the ISD and the original error. In general you may want to avoid printing the ISD. If I left that in it can be removed.